pvp/common/message.py

```diff
@@ -116,7 +113,8 @@ def __setitem__(self, key, value):
         elif key.lower() in self.additional_values:
             return setattr(self, key.lower(), value)
         else:
-            raise KeyError(f'No such value as {key}')
```

Comment:
would need a bit more discussion on these changes -- these are in place to make sure we have a tight and uniform values api and i'm not sure what removing them adds

Reply:
same here. This is stuff that we tested doesn't happen in the current version of the code. We could leave this in for people who fiddle with the API - but do we want that? When in doubt, I felt like removing code that is not used